tests/containers_test.py

```diff
@@ -65,6 +66,37 @@ def test_document_boolean_visibility_invalid(self) -> None:
         assert d.features[0].visibility is None
         assert d.features[0].isopen
 
+    def test_container_creation(self) -> None:
+        container = containers._Container(
+            ns="ns",
+            id="id",
+            target_id="target_id",
+            name="name"
+        )
+        assert container.ns == "ns"
+        assert container.name == "name"
+
+    def test_container_feature_append(self) -> None:
+        container = containers._Container(
+            ns="ns",
+            id="id",
+            target_id="target_id",
+            name="name"
+        )
+        feature = features._Feature(name="new_feature")
+        assert container.append(feature) is None
```

Comment:
I don't think this line is needed, as `.append` always returns `None`

Reply:
changed